packages/synapse-react-client/src/lib/containers/CreateProjectModal.tsx

```diff
@@ -40,55 +41,42 @@ export const CreateProjectModal: React.FunctionComponent<
       }
     }
   }
-  return (
+
+  const dialogContent = (
     <>
-      <Modal
-        className="CreateProjectModal bootstrap-4-backport"
-        show={isShowingModal}
-        animation={false}
-        onHide={hide}
-        backdrop="static"
-      >
-        <Modal.Header closeButton>
-          <Modal.Title>Create a new Project</Modal.Title>
-        </Modal.Header>
-        <Modal.Body>
-          <Form.Group>
-            <Form.Label htmlFor={'projectInput'}>Project Name</Form.Label>
-            <Form.Control
-              id="projectInput"
-              data-testid="projectInput"
-              value={projectName}
-              onChange={event => {
-                setProjectName(event.target.value)
-              }}
-              onKeyDown={(event: any) => {
-                if (event.key === 'Enter') {
-                  if (projectName !== '') {
-                    onCreateProject()
-                  }
-                }
-              }}
-            />
-          </Form.Group>
-          {errorMessage && <Alert variant="danger">{errorMessage}</Alert>}
-        </Modal.Body>
-        <Modal.Footer>
-          <div className="ButtonContainer">
-            <Button variant="default" onClick={onClose}>
-              Cancel
-            </Button>
-            <Button
-              variant="primary"
-              onClick={() => {
+      <TextField
+        id="projectInput"
+        label="Project Name"
+        value={projectName}
+        fullWidth
+        onChange={event => {
+          setProjectName(event.target.value)
+        }}
+        inputProps={{
+          onKeyDown: (event: React.KeyboardEvent<HTMLInputElement>) => {
```

Comment:
TextField doesn't expose `onKeyDown` as a prop, so pass to Input via `inputProps`, see [here](https://github.com/mui/material-ui/issues/12946#issuecomment-423850042)